packages/editor-ui/src/composables/useRunWorkflow.ts

```diff
@@ -37,6 +37,24 @@ import { get } from 'lodash-es';
 import { useExecutionsStore } from '@/stores/executions.store';
 import { useLocalStorage } from '@vueuse/core';
 
+const getDirtyNodesIds = (runData: IRunData): string[] | undefined => {
+	const workflowsStore = useWorkflowsStore();
+
+	const dirtyIds = Object.entries(runData).reduce<string[]>((acc, [nodeName, tasks]) => {
+		if (!tasks.length) return acc;
+
+		const updatedAt = workflowsStore.getParametersLastUpdate(nodeName) ?? 0;
+
+		if (updatedAt > tasks[0].startTime) {
+			acc.push(nodeName);
+		}
```

Comment:
@r00gm could we have issues with timezones here?

Reply:
maybe, but do we have any other way to compare the dates?

Reply:
We're using `Date.now()` which returns UTC. So all good.